webapp/src/components/ActivityPage/ActivityPage.js

```diff
@@ -42,10 +41,22 @@ export default class ActivityPage extends React.PureComponent {
   }
 
   renderTransactionLists() {
-    const { pendingTransactions, transactionHistory, network } = this.props
+    const {
+      pendingTransactions,
+      transactionHistory,
+      network,
+      onClear,
+      address
+    } = this.props
 
     return (
       <React.Fragment>
+        <div className="clear-button">
+          <Button size="tiny" className="link" onClick={() => onClear(address)}>
```

Comment:
We tend to avoid creating new functions on render, more info [here](https://github.com/decentraland/standards/blob/master/standards/react-redux.md#32-avoid-creating-stuff-inside-the-render-function)